packages/transformers/js/src/visitors/dependencies.js

```diff
@@ -188,8 +216,43 @@ export default ({
           meta: {
             webworker: true,
           },
-        });
-        return;
+        };
+
+        if (isStringLiteral(args[0])) {
+          addURLDependency(asset, ast, args[0], opts);
```

Comment:
Maybe we could add a warning here? IMO we should probably remove it before stable parcel 2, so a deprecation warning in the meantime would be good.

Reply:
![Bildschirmfoto 2021-02-11 um 19 30 43](https://user-images.githubusercontent.com/4586894/107681673-a512ba80-6c9f-11eb-8379-9bdf9abba576.png)